sockeye/training.py

```diff
@@ -613,7 +614,7 @@ def _load_params(self, output_folder: str, checkpoint: int):
         """
```

Comment:
can you update the comment in line 613? `Loads parameters from disk, sets self.params and module's parameters.`